src/common/integrations/protocolIntegrationPicked.ts

```diff
@@ -113,5 +114,42 @@ export const protocolIntegrationPicked = async (
     }
 
     navigate("/config/zha/add");
+  } else if (domain === "matter") {
+    const entries = await getConfigEntries(hass, {
+      domain,
+    });
+    if (!isComponentLoaded(hass, domain) || !entries.length) {
+      // If the component isn't loaded, ask them to load the integration first
+      showConfirmationDialog(element, {
+        title: hass.localize(
+          "ui.panel.config.integrations.config_flow.missing_zwave_zigbee_title",
```

Comment:
And that's why we shouldn't have implementation details in translation strings 😬

Reply:
😅